assets/airports/eddf.json

```diff
@@ -6,940 +6,4866 @@
     "rr_radius_nm": 5.0,
-    "rr_center": ["N50d01m18.38", "E8d33m23.24"],
+    "rr_center": ["N50d01m18.38", "E008d33m23.24"],
     "has_terrain": true,
     "wind": {
-        "angle": 230,
-        "speed": 5
+        "angle": 202,
+        "speed": 8
     },
-    "arrivalRunway": "25L",
-    "departureRunway": "25L",
+    "arrivalRunway": "25R",
+    "departureRunway": "25C",
     "airspace": [
         {
             "floor": 0,
             "ceiling": 135,
             "airspace_class": "C",
             "poly": [
-                ["N050d17m10.000", "E009d37m20.000"],
-                ["N049d58m39.000", "E009d41m33.000"],
-                ["N049d55m27.000", "E009d42m17.000"],
-                ["N049d50m35.000", "E009d36m48.000"],
-                ["N049d46m32.000", "E009d32m15.000"],
-                ["N049d45m20.000", "E009d30m54.000"],
-                ["N049d49m22.000", "E009d14m50.000"],
-                ["N049d49m46.000", "E009d12m19.000"],
-                ["N049d46m08.000", "E009d11m07.000"],
-                ["N049d44m00.000", "E09d08m01.000"],
-                ["N049d43m51.000", "E09d06m50.000"],
-                ["N049d42m58.000", "E08d57m38.000"],
-                ["N049d41m47.000", "E08d46m37.000"],
-                ["N049d40m52.000", "E08d29m28.000"],
-                ["N049d40m41.000", "E08d26m16.000"],
-                ["N049d40m15.000", "E08d18m30.000"],
-                ["N049d39m46.000", "E08d10m03.000"],
-                ["N049d43m42.000", "E07d59m09.000"],
-                ["N049d44m51.000", "E07d55m58.000"],
-                ["N049d46m24.000", "E07d51m40.000"],
-                ["N049d48m51.000", "E07d44m46.000"],
-                ["N049d49m47.000", "E07d42m10.000"],
-                ["N049d49m56.000", "E07d41m45.000"],
-                ["N049d56m33.000", "E07d41m28.000"],
-                ["N050d02m15.000", "E007d41m13.000"],
-                ["N050d02m37.000", "E007d41m12.000"],
-                ["N050d11m39.000", "E007d40m49.000"],
-                ["N050d11m49.000", "E007d41m38.000"],
-                ["N050d12m00.000", "E007d42m33.000"],
-                ["N050d13m57.000", "E007d52m18.000"],
-                ["N050d15m45.000", "E008d01m55.000"],
-                ["N050d17m51.000", "E008d12m01.000"],
-                ["N050d19m02.000", "E008d17m59.000"],
-                ["N050d19m46.000", "E008d21m49.000"],
-                ["N050d20m46.000", "E008d26m55.000"],
-                ["N050d22m37.000", "E008d36m33.000"],
-                ["N050d24m06.000", "E008d44m13.000"],
-                ["N050d25m53.000", "E008d53m44.000"],
-                ["N050d27m43.000", "E009d03m24.000"],
-                ["N050d30m46.000", "E009d19m48.000"],
-                ["N050d31m13.000", "E009d22m15.000"],
-                ["N050d30m39.000", "E009d29m10.000"],
-                ["N050d27m12.000", "E009d35m01.000"],
-                ["N050d25m25.000", "E009d35m26.000"]
+                ["N50d11m39", "E007d40m49"],
```

Comment:
The airspace you've added is actually the class C airspace around Frankfurt, not the approach airspace. Think of the DCA class B airspace and the actual TRACON; they're different. In the case of Germany they're usually pretty similar but not the same. the diagram [here](https://www.ivao.de/eddf-app/) shows the approach airspace. You might find the coordinates for it in sector files

Reply:
disregard